app/src/main/java/org/sopt/and/data/di/NetworkModule.kt

```diff
@@ -0,0 +1,86 @@
+package org.sopt.and.data.di
+
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.Json
+import okhttp3.MediaType.Companion.toMediaTypeOrNull
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import org.sopt.and.BuildConfig
+import org.sopt.and.data.interceptor.AccessTokenInterceptor
+import org.sopt.and.utils.TokenManager
+import retrofit2.Retrofit
+import java.util.concurrent.TimeUnit
+import javax.inject.Qualifier
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object NetworkModule {
+
+    @Qualifier
+    @Retention(AnnotationRetention.BINARY)
+    annotation class MainServer
+
+    @Qualifier
+    @Retention(AnnotationRetention.BINARY)
+    annotation class Token
+
+    @Provides
+    @Singleton
+    @OptIn(ExperimentalSerializationApi::class)
+    fun provideJson(): Json = Json {
+        isLenient = true
+        prettyPrint = true
+        explicitNulls = false
+        ignoreUnknownKeys = true
+    }
+
+    @Provides
+    @Singleton
+    fun provideHttpLoggingInterceptor(): HttpLoggingInterceptor {
+        return HttpLoggingInterceptor().apply {
+            level = HttpLoggingInterceptor.Level.BODY
+        }
+    }
+
+    @Provides
+    @Singleton
+    @Token
+    fun provideTokenInterceptor(tokenManager: TokenManager): AccessTokenInterceptor {
+        return AccessTokenInterceptor(tokenManager)
+    }
+
+    @Provides
+    @Singleton
+    fun provideOKHttpClient(
+        httpLoggingInterceptor: HttpLoggingInterceptor,
+        accessTokenInterceptor: AccessTokenInterceptor
+    ): OkHttpClient =
+        OkHttpClient.Builder().apply {
+            connectTimeout(30, TimeUnit.SECONDS)
+            readTimeout(30, TimeUnit.SECONDS)
+            writeTimeout(30, TimeUnit.SECONDS)
+            addInterceptor(httpLoggingInterceptor)
```

Comment:
실제 릴리즈 코드에 포함될 이유는 없으므로 디버그 빌드에만 추가하도록 지정함은 어떨지요?
```suggestion
            if (BuildConfig.DEBUG) addInterceptor(httpLoggingInterceptor)
```

Reply:
디버그 빌드에 따로 지정가능한지 몰랐는데 감사합니다.